test/fit_tests/common/fit_common.py

```diff
@@ -0,0 +1,528 @@
+        else:
+            STACK_CONFIG.update(detailfile)
+
+
+# This section derives default stack configuration data from STACK-CONFIG, use environment to override
+ARGS_LIST.update(
+    {
+        "usr": GLOBAL_CONFIG['credentials']['ora'][0]['username'],
+        "pwd": GLOBAL_CONFIG['credentials']['ora'][0]['password']
+    }
+)
+
+if ARGS_LIST["stack"] != "None":
+    if "ora" in STACK_CONFIG[ARGS_LIST["stack"]]:
+        ARGS_LIST["ora"] = STACK_CONFIG[ARGS_LIST["stack"]]['ora']
+    else:
+        ARGS_LIST["ora"] = "localhost"
+    if "bmc" in STACK_CONFIG[ARGS_LIST["stack"]]:
+        ARGS_LIST["bmc"] = STACK_CONFIG[ARGS_LIST["stack"]]['bmc']
+    if "hyper" in STACK_CONFIG[ARGS_LIST["stack"]]:
+        ARGS_LIST["hyper"] = STACK_CONFIG[ARGS_LIST["stack"]]['hyper']
+
+# set api port and protocol
+if ARGS_LIST['http'] == "True":
+    API_PROTOCOL = 'http'
+    API_PORT = GLOBAL_CONFIG['ports']['http']
+if ARGS_LIST['https'] == "True":
+    API_PROTOCOL = 'https'
+    API_PORT = GLOBAL_CONFIG['ports']['http']
+if ARGS_LIST["ora"] == "localhost":
+    API_PROTOCOL = 'http'
+    API_PORT = '8080'
+
+def timestamp(): # return formatted current timestamp
+    return time.strftime("%Y-%m-%dT%H:%M:%S", time.gmtime())
+
+# This routine executes a sleep with countdown
+def countdown(sleep_time, sleep_interval=1):
+    sys.stdout.write("Sleeping for " + str(sleep_time * sleep_interval)
+                     + " seconds.")
+    sys.stdout.flush()
+    for _ in range(0, sleep_time):
+        time.sleep(sleep_interval)
+        sys.stdout.write(".")
+        sys.stdout.flush()
+    print "Waking!"
+    return
+
+def remote_shell(shell_cmd, expect_receive="", expect_send="", timeout=300, address=ARGS_LIST['ora'], user=ARGS_LIST['usr'], password=ARGS_LIST['pwd']):
+    '''
+    Run ssh based shell command on a remote machine at ARGS_LIST["ora"]
+
+    :param shell_cmd: string based command
+    :param expect_receive:
+    :param expect_send:
+    :param timeout: in seconds
+    :param address: IP or hostname of remote host
+    :param user: username of remote host
+    :param password: password of remote host
+    :return: dict = {'stdout': str:ouput, 'exitcode': return code}
+    '''
+
+    # if localhost just run the command local
+    if ARGS_LIST['ora'] == 'localhost':
+        (command_output, exitstatus) = \
+            pexpect.run("sudo bash -c \\\"" + shell_cmd + "\\\"",
+                        withexitstatus=1,
+                        events={"assword": password + "\n"},
+                        timeout=timeout, logfile=logfile_redirect)
+        return {'stdout':command_output, 'exitcode':exitstatus}
+
+    # this clears the ssh key from ~/.ssh/known_hosts
+    subprocess.call(["touch ~/.ssh/known_hosts;ssh-keygen -R "
+                     + address  + " -f ~/.ssh/known_hosts >/dev/null 2>&1"], shell=True)
+
+    logfile_redirect = None
+    shell_cmd.replace("'", "\\\'")
+    if VERBOSITY >= 4:
```

Comment:
I see a lot judgement of `if VERBOSITY >= 4` and `if VERBOSITY >= 9`, It's not self-explained, change them to be more descriptive will be better.